hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/TestReconUtils.java

```diff
@@ -44,46 +47,37 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.hadoop.hdds.conf.OzoneConfiguration;
 import org.apache.hadoop.hdfs.web.URLConnectionFactory;
-import org.junit.Assert;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 /**
  * Test Recon Utility methods.
  */
 public class TestReconUtils {
 
-  @Rule
-  public TemporaryFolder folder = new TemporaryFolder();
+  @TempDir
+  private static Path temporaryFolder;
 
   @Test
-  public void testGetReconDbDir() throws Exception {
+  public void testGetReconDbDir(@TempDir File newDir) throws Exception {
```

Comment:
Can use member `temporaryFolder` instead of param in this method?

Reply:
Done, Thanks.